LDMP/gui/WidgetMain.ui

```diff
@@ -93,20 +102,29 @@
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_import">
+            <property name="toolTip">
+             <string>Add datasets from a local location</string>
```

Comment:
The button's label says `Import` so I guess the tooltip should also contain that word, instead of `Add`
```suggestion
             <string>Import external datasets from a local directory into Trends.Earth</string>
```